docs/.vitepress/config.mjs

```diff
@@ -17,9 +18,89 @@ export default defineConfig({
   srcDir: "docs",
   lastUpdated: true,
   locales: {
-    root: {
+    en: {
```

Comment:
I wonder if it would have been better to keep English as the root to keep the URLs nice and tidy for the majority of readers. But we can also keep it as-is, for example, GitHub docs also include `en` if you have English selected: https://docs.github.com/en

Reply:
The structure is a bit influenced by the translation tools, which expect wildcard of the style `.../{locale}/**/*.md`. If we break that convention with one of the languages, we might not be able to configure the translation pipeline, or it'll become a bit more inconvenient.